### DISCUSSION TITLE
Help on hotrc HT-6A please

### DISCUSSION THREAD

**Discussion by MrVitorF** [pascallanger/DIY-Multiprotocol-TX-Module#821] [Q&A] [answered]
Hi, anyone has any idea if and how i can bind to the crappy hotrc HT-6A?
They say it has GFSK Modulation and Spread Spectrum FHSS, but i tried a few and no bind.
Any tip on how to bind to dis receiver would be mutch apreciated, thank you.

**Comment by pascallanger** [comment, marked as answer]
Never seen this manufacturer before... I would need the TX + RX to add the protocol.

**Reply by MrVitorF** [reply]
Thank you anyway, no clue how to get that.
Don't worry about it, let me just say that you have my most profound respect for what you are doing. thank you.

**Comment by Dietrich-L** [comment]
Any news on this topic?
The HotRC products seem to become more popular.
